src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1201,6 +1215,19 @@ public Builder recordProcessorFactory(IRecordProcessorFactory recordProcessorFac
             return this;
         }
 
+        /**
+         * Provide a {@link LeaseSelector
+         * LeaseSelector}.
+         *
+         * @param leaseSelector
+         *            Used to get fetch the list of leases that can be taken
+         * @return A reference to this updated object so that method calls can be chained together.
+         */
+        public Builder leaseSelector(LeaseSelector<KinesisClientLease> leaseSelector) {
```

Comment:
You have the `@Setter` annotation on the variable. You should not need to create this method.

Reply:
Done